src/components/form/AddNewBook.js

```diff
@@ -21,31 +23,24 @@ function AddNewBook() {
     const newBook = {
       item_id: uuidv4(),
       title,
-      category,
+      category: 'Action',
     };
-
     dispatch(postBook(newBook));
 
     setCategory('');
     setTitle('');
   };
 
   return (
-    <form>
-      <input
-        type="text"
-        placeholder="Title"
-        value={title}
-        onChange={handleTitle}
-      />
-      <input
-        type="text"
-        placeholder="Category"
-        value={category}
-        onChange={handleCategory}
-      />
-
-      <button type="submit" onClick={submitBookToStore}>
+    <form className="flex justify-between py-8 px-28 max-w-7xl mx-auto md:flex items-center space-x-4 space-y-4 md:space-y-0">
+      <Input title={title} handleTitle={handleTitle} />
+      <Example handleCategory={handleCategory} category={category} />
+
+      <button
+        type="submit"
+        onClick={submitBookToStore}
+        className="bg-blue-400 text-white px-4 py-2"
+      >
         Add Book
       </button>
     </form>
```

Comment:
As required, Styling is should be the same as the one provided on [Zeplin](https://zeplin.io/)

Your design
> ![Screenshot (360)](https://user-images.githubusercontent.com/66112760/150649469-9f7899c0-bdc0-4d30-8428-ff21b57be72a.png)


Zeplin design 
>  ![Screenshot (360)](https://user-images.githubusercontent.com/64774936/150636616-41852bf1-d645-4201-a56d-6d0625bafe4a.png) 

Kindly work on the form section of the webpage 👍